modules/op-configuring-rbac-trusted-ca-flags.adoc

```diff
@@ -0,0 +1,46 @@
+// This module is included in the following assemblies:
+// * install_config/customizing-configurations-in-the-tektonconfig-cr.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="op-configuring-rbac-trusted-ca-flags.adoc_{context}"]
+= Configuring RBAC and Trusted CA flags
+
+The {pipelines-title} Operator provides independent control over RBAC resource creation and Trusted CA bundle config map through two separate flags, `createRbacResource` and `createCABundleConfigMaps`.
+
+[cols="1,3,1", options="header"]
+|===
+|Parameter |Description |Default value
+
+|`createRbacResource`
+|Controls the creation of RBAC resources only. This flag does not affect Trusted CA bundle config map.
+|`true`
+
+|`createCABundleConfigMaps`
+|Controls the creation of Trusted CA bundle config map. This flag must be set to `false` to disable config map creation.
```

Comment:
controls the creation of Trusted CA bundle config map and Service CA bundle config map

Reply:
Fixed.